entries/resizable.xml

```diff
@@ -153,12 +153,24 @@
 					<desc>The size represented as <code>{ width, height }</code> before the resizable is resized</desc>
 				</property>
 				<property name="position" type="Object">
-					<desc>The current position represented as <code>{ left, top }</code></desc>
+					<desc>The current position represented as <code>{ left, top }</code>. The values may be changed to modify where the element will be positioned. Useful for custom resizing logic.</desc>
 				</property>
 				<property name="size" type="Object">
-					<desc>The current size represented as <code>{ width, height }</code></desc>
+					<desc>The current size represented as <code>{ width, height }</code>. The values may be changed to modify where the element will be positioned. Useful for custom resizing logic.</desc>
 				</property>
 			</argument>
+			<example>
+				<desc>Constrain horizontal resizing:</desc>
```

Comment:
Well, that's what you already have. Could add my example in addition to this one - if you think that is still too distracting, leave it as-is.